extension/js/common/core/common.ts

```diff
@@ -84,8 +84,9 @@ export class Str {
     if (email.indexOf(' ') !== -1) {
       return false;
     }
-    // for MOCK tests, we need emails like me@domain.com:8001 to pass
+    // for MOCK tests, we need emails like me@domain.com:8001 or me@localhost:8001to pass
```

Comment:
do we still need these updates?

Reply:
indeed, should not be needed

Reply:
We still need the original `:8001` removal because many account emails for tests are written that way. I've filed #4631